circuit_maintenance_parser/parser.py

```diff
@@ -349,10 +349,18 @@ def get_key_with_string(dictionary: dict, string: str):
 
     def get_llm_question(self):
         """Return the LLM question."""
-        custom_llm_question = os.getenv("LLM_QUESTION")
+        custom_llm_question = os.getenv("PARSER_LLM_QUESTION_STR")
```

Comment:
What if we have multiple parsers? Like I have subscriptions to WatsonX, ChatGPT, and Bard?

Reply:
The LLM question is the same for all